diff --git a/node_modules/jsmind/js/jsmind.js b/node_modules/jsmind/js/jsmind.js
index be0f0b2..5376af0 100644
--- a/node_modules/jsmind/js/jsmind.js
+++ b/node_modules/jsmind/js/jsmind.js
@@ -2742,6 +2742,9 @@
         },
 
         handler : function(e){
+            if (e.which == 9) {
+                e.preventDefault();
+            } //prevent tab to change focus in browser
             if(this.jm.view.is_editing()){return;}
             var evt = e || event;
             if(!this.opts.enable){return true;}
